thirdparty/google-styleguide/run_linter.sh

```diff
@@ -40,5 +40,7 @@ done
 
 HEADERS=`find $INCLUDE_DIRS -name '*.h' | sort | uniq | tr '\n' ' '`
 SOURCES=`find $SOURCE_DIRS -name  '*.cpp' | sort | uniq | tr '\n' ' '`
-REPOSITORY="$(realpath --physical "$(dirname "$0")/../..")"
+# this realpath alternative should work on mac
+alias prealpath="python -c 'import os, sys; print(os.path.realpath(sys.argv[1]))'"
```

Comment:
where is this alias used?

Reply:
In the end I incorporated it to the below command, because the alias didn't work in subshells. I forgot to remove this one.

Reply:
removed the leftover line in 0c23d4e